targets/tinyfpga_bx/base.py

```diff
@@ -118,7 +113,6 @@ def __init__(self, platform, **kwargs):
         # template anyway just in case.
         # Disable final deep-sleep power down so firmware words are loaded
         # onto softcore's address bus.
-        platform.toolchain.build_template[3] = "icepack -s {build_name}.txt {build_name}.bin"
-        platform.toolchain.nextpnr_build_template[2] = "icepack -s {build_name}.txt {build_name}.bin"
+        platform.toolchain.build_template[2] = "icepack -s {build_name}.txt {build_name}.bin"
```

Comment:
This should call `platform_toolchain_extend` I think?

Reply:
Also IIRC the litex defaults for those changed in a useful way, so it might not even be necessary to override them any longer (just as the nextpnr override wasn't necessary any more as it's now the default).  In my WIP on the TinyFPGA BX (see https://github.com/timvideos/litex-buildenv/pull/348#issuecomment-604296404) I just removed both of them, and it seemed to be working the same as before.

(Same comment applies to several points above, but just posting once for brievity :-) )

Reply:
@ewenmcneill You are right. I also noticed that `-s` switch to `icepack` is now by default in LiteX. 

I verified the content of `build_template` before and after modifying it and can confirm that the above line does not change anything and is not necessary anymore. I'll remove it from all affected files.